source/Engine.h

```diff
@@ -252,8 +252,10 @@ class Engine {
 	TestContext *testContext = nullptr;
 
 	double zoom = 1.;
+	double trueZoom = 1.;
```

Comment:
trueZoom is not exactly precise naming as the zoom the player experiences is another

Reply:
renamed to baseZoom

Reply:
thank you